Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -120,16 +122,18 @@ listDriverRides driverId mbLimit mbOffset mbOnlyActive mbRideStatus = do
     rideDetail <- runInReplica $ QRD.findById ride.id >>= fromMaybeM (VehicleNotFound driverId.getId)
     rideRating <- runInReplica $ QR.findRatingForRide ride.id
     driverNumber <- RD.getDriverNumber rideDetail
-    pure $ mkDriverRideRes rideDetail driverNumber rideRating (ride, booking)
+    mbExophone <- CQExophone.findByPrimaryPhone booking.primaryExophone
```

Comment:
`booking.primaryExophone` is not `Maybe`. I'm just don't know what to do if this exophone was deleted from DB via dashboard or manually? What we should show to user? Throw an error will be not properly I think, because it will broke our flow. Can we make `DriverRideRes.exoPhone` optional?